command/README.md

```diff
@@ -0,0 +1,51 @@
+<div align="center">
+  <h1>expo-github-action/command</h1>
+  <p>Run <a href="https://github.com/expo/expo-cli">Expo CLI</a> or <a href="https://github.com/expo/eas-cli">EAS CLI</a> command by a comment on the PR</p>
+  <p>
+    <a href="https://github.com/expo/expo-github-action/releases">
+      <img src="https://img.shields.io/github/v/release/expo/expo-github-action" alt="releases" />
+    </a>
+    <a href="https://github.com/expo/expo-github-action/actions">
+      <img src="https://img.shields.io/github/workflow/status/expo/expo-github-action/test" alt="builds" />
+    </a>
+    <a href="https://github.com/expo/expo-github-action/blob/main/LICENSE.md">
+      <img src="https://img.shields.io/github/license/expo/expo-github-action" alt="license" />
+    </a>
+  </p>
+  <p align="center">
+    <a href="#configuration-options"><b>Usage</b></a>
+    &nbsp;&nbsp;&mdash;&nbsp;&nbsp;
+    <a href="#example-workflows"><b>Examples</b></a>
+    &nbsp;&nbsp;&mdash;&nbsp;&nbsp;
+    <a href="#things-to-know"><b>Caveats</b></a>
+    &nbsp;&nbsp;&mdash;&nbsp;&nbsp;
+    <a href="https://github.com/expo/expo-github-action/blob/main/CHANGELOG.md"><b>Changelog</b></a>
+  </p>
+</div>
+
+## Configuration options
+
+This action is customizable through variables defined in the [`action.yml`](action.yml).
+Here is a summary of all the input options you can use.
+
+| variable       | default                     | description                                                                                      |
+| -------------- | --------------------------- | ------------------------------------------------------------------------------------------------ |
+| **project**    | -                           | The relative path to the Expo project                                                            |
+| **reaction**    | -                   | If set, the specified emoji "reaction" is put on the comment to indicate that the trigger was detected. For example, "rocket". |
```

Comment:
I'm not sure how beneficial this property would be for users. I think just replying with a 👍 should be understandable that the command is understood and picked up 😄